eisbuk-admin/cypress/plugins/firebasePlugin/updateFirestore/index.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from "path";
+
+import { TaskHandler } from "../types";
+
+import readTestData from "./readTestData";
+import updateFirestoreData from "./updateFirestoreData";
+
+interface Payload {
+  organization: string;
+  files: string[];
+}
+
+const handleFirestoreUpdate: TaskHandler<Payload> = async ({
+  organization,
+  files,
+}) => {
+  const testFilesDir = path.join(__dirname, "..", "..", "..", "__testData__");
```

Comment:
Thanks for not using the current working directory! ;-)

Reply:
Haha, np...this will also be read from the config.